app/controllers/albums.js

```diff
@@ -0,0 +1,15 @@
+const albumsService = require('../services/album');
+
+module.exports.albums = (_, res, next) => {
+  albumsService
+    .albums()
+    .then(albumsList => albumsList)
+    .catch(error => next(error));
+};
+
+module.exports.photos = (albumId, _, res, next) => {
+  albumsService
+    .photos(albumId)
```

Comment:
What are you trying to do here?

Unless the first param is an error, which `albumId` is not, express functions should just have `(req, res, next)`
If you want to access the album  `id` param in `/albums/:id/photos`, you get it from `req.params.id`. Express automatically maps it from the url.